plugin.yml

```diff
@@ -143,4 +144,10 @@ permissions:
         default: true
     populationdensity.unlimitedhoppers:
         description: Players with this permission node ignore the hoppers-per-chunk limit defined in the config file.
-        default: op
\ No newline at end of file
+        default: op
+    populationdensity.randomregion:
+        description: Teleports the player to a random region.
+        default: op
+    populationdensity.lag:
+        description: Provides detailed server statistic information.
+        default: op
```

Comment:
Thank you! However, both of these should default to `true` since that a) retains existing behavior and b) the /lag command is also run if someone just says "lag" in chat.